From 9db7867c9e7188348df7aa5ee9d2763c63559bf6 Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Fri, 31 Jul 2020 17:56:37 +0700
Subject: [PATCH 423/504] ARM: dts: r8a7742: Add vspd{0,1} description to SoC
 dtsi

This patch adds vspd0 and vspd1 (VSP for DU) support for R8A7742.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7742.dtsi | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7742.dtsi b/arch/arm/boot/dts/r8a7742.dtsi
index cd7162b..1d26f53 100644
--- a/arch/arm/boot/dts/r8a7742.dtsi
+++ b/arch/arm/boot/dts/r8a7742.dtsi
@@ -118,6 +118,22 @@
 		};
 	};
 
+	vspd0: vspd0@fe930000 {
+		compatible = "renesas,vsp2";
+		reg = <0 0xfe930000 0 0x7404>;
+		interrupts = <0 246 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&mstp1_clks R8A7742_CLK_VSP1_DU0>;
+		power-domains = <&sysc R8A7742_PD_ALWAYS_ON>;
+	};
+
+	vspd1: vspd1@fe938000 {
+		compatible = "renesas,vsp2";
+		reg = <0 0xfe938000 0 0x7404>;
+		interrupts = <0 247 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&mstp1_clks R8A7742_CLK_VSP1_DU1>;
+		power-domains = <&sysc R8A7742_PD_ALWAYS_ON>;
+	};
+
 	du: display@feb00000 {
 		compatible = "renesas,du-r8a7742";
 		reg = <0 0xfeb00000 0 0x70000>,
-- 
2.7.4

